.github/workflows/continuous-integration.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
       - types
     runs-on: ubuntu-latest
     steps:
-      - name: Merge me!
-        uses: ridedott/merge-me-action@master
+      - name: Auto merge
+        uses: ridedott/dependabot-auto-merge-action@master
         with:
```

Comment:
We should specify a new bot name.

Reply:
This is taken from the firestore-schema repository. Which I assumed would have a correct implementation of this. If this changed. I'm looking at https://github.com/ridedott/merge-me-action.

I see nothing wrong about how this was setup before I made these changes.

Reply:
We made a switch from dependabot-preview to dependabot, so we must use https://github.com/ridedott/merge-me-action#enable-auto-merge-for-a-different-bot. See https://github.com/ridedott/eslint-config/blob/master/.github/workflows/continuous-integration.yaml#L100.

Reply:
I reverted the changes and updated the configuration. Reading [this](https://github.com/ridedott/merge-me-action#enable-auto-merge-for-a-different-bot) documentation and looking at Dependabot's PR's, it is not really clear that the username is wrong.

At first glance, it looks as if Dependabot's PR's are authored by `dependabot`, without `bot`. `bot` does not seem to be part of the username.